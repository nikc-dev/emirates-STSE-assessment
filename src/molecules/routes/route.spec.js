import React from 'react';
import { mount, shallow, configure } from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import { createSerializer } from 'enzyme-to-json';
import toJson from 'enzyme-to-json';
import Route from './route';

configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

it('should render Page Component correctly', () => {
  const component = shallow(
    <Route to={`test`} from={`test`} departsAt={`test`} buttonId={1} arrivesAt={`test`} stops={`test`} booked={true}/>,
  );
  expect(toJson(component)).toMatchSnapshot();
});




