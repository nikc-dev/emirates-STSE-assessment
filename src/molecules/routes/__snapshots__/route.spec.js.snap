// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Page Component correctly 1`] = `
<div
  className="route"
>
  <Container
    fluid={false}
  >
    <Row
      className="route__row-booked"
      noGutters={false}
    >
      <Col
        className="route__ek-icon"
      >
        <img
          alt="express"
          className="route__ek-img"
          src="ek.png"
        />
      </Col>
      <Col
        className="route__col"
      >
        <div
          className="route__time-container"
        >
          <p
            className="route__time"
          >
            test - test
          </p>
          <p
            className="route__carrier"
          >
            Emirates
          </p>
        </div>
      </Col>
      <Col
        className="route__col"
      >
        <div
          className="route__destination-container"
        >
          <p
            className="route__destination"
          >
            test-test
          </p>
          <p
            className="route__stops"
          >
            test
          </p>
        </div>
      </Col>
      <Col
        className="route__col"
      >
        <div
          className="route__select-cta-container"
        >
          <p
            className="route__selected"
          >
            Selected
          </p>
        </div>
      </Col>
    </Row>
    <hr
      className="route__hr"
    />
  </Container>
</div>
`;
